.github/workflows/tests.yml

```diff
@@ -167,7 +167,7 @@ jobs:
       with:
         ref: ${{ github.event.pull_request.head.sha }}
     - name: Run ShellCheck
-      uses: ludeeus/action-shellcheck@1.1.0
+      uses: ludeeus/action-shellcheck@master
```

Comment:
Would it be possible to refer to a specific commit instead? E.g. the last one from the default branch? This way we will prevent unexpected breakages if they change anything on that branch?

Reply:
Alternatively we can use different shellcheck action, but I don't like using `master`... 🙄

Reply:
Thanks @thozza. Latest commit works.